moveit_msgs/RobotTrajectory trajectory_in
float64 velocity_scaling
---
moveit_msgs/RobotTrajectory trajectory_out